### DISCUSSION TITLE
Reincode audio in srt stream via ffmpeg

### DISCUSSION THREAD

**Discussion by flowless911-1** [bluenviron/mediamtx#3877] [Questions]
### Question

Hi. I stream srt to a website where is webrtc output. Can i re-encode audiocodec aac in srt stream to opus? 
Mediamtx 1.9.2
Im get an error if i use config for rtsp in tutorial:

paths:
  new:
  original:
    runOnReady: >
      ffmpeg -i srt://localhost:8585/original  -vcodec copy -c:a libopus  -f srt srt://localhost:8585/new
    runOnReadyRestart: yes

ffmpeg version 4.4.2-0ubuntu0.22.04.1 Copyright (c) 2000-2021 the FFmpeg developers
 ----- $ffmpeg info -----
2024/10/14 16:48:28 INF [SRT] [conn 127.0.0.1:32772] opened
2024/10/14 16:48:28 INF [SRT] [conn 127.0.0.1:32772] closed: invalid stream ID '': stream ID must be 'action:pathname[:query]' or 'action:pathname:user:pass[:query]', where action is either read or publish, pathname is the path name, user and pass are the credentials, query is an optional token containing additional information
16:48:28.176906/SRT:RcvQ:w1!W:SRT.cn: processAsyncConnectRequest: REJECT reported from HS processing: Peer rejected connection - not processing further
[srt @ 0x564dcf265340] Connection to srt://localhost:8585/original failed: Input/output error
srt://localhost:8585/original: Input/output error
2024/10/14 16:48:28 INF [path original] runOnReady command exited: command exited with code 0
2024/10/14 16:48:29 INF [path original] runOnReady command stopped

**Comment by aler9** [comment]
URLs `srt://localhost:8585/original` and `srt://localhost:8585/new` are not valid SRT URLs, take a look at the README for instructions on how to format SRT URLs:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#by-protocol
